shared/src/maintenance.rs

```diff
@@ -19,7 +19,7 @@ impl Maintaining for ServiceMaintenance {
     async fn run_maintenance(&self) -> Result<()> {
         for result in join_all(self.maintainers.iter().map(|m| m.run_maintenance())).await {
             if let Err(err) = result {
-                tracing::error!("failed with: {}", err);
+                tracing::error!("Service Maintenance Error: {:?}", err);
```

Comment:
As far as my understanding goes, this is the key change, which will make us log the entire error chain instead of just the top level context.

Reply:
Good to know!